test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -434,3 +467,85 @@ test.requestHooks(logger, enrollViaSmsVersionUpgradeMocksGoBack)('should not sho
   await t.expect(errorBox.exists).notOk;
 });
 
+test.requestHooks(logger, enrollViaQRcodeEnableBiometricsMocks)('should see ov enable biometrics message during enroll via QR code', async t => {
+  const enrollOktaVerifyPage = await setup(t);
+  await t.expect(enrollOktaVerifyPage.getFormTitle()).eql('Set up Okta Verify');
+  await t.expect(enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(true);
+  await t.expect(enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(false);
+  await t.expect(enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(false);
+  await t.expect(enrollOktaVerifyPage.hasQRcode()).eql(true);
+  await t.expect(enrollOktaVerifyPage.getQRInstruction()).eql(qrCodeInstruction);
+  const errorBox = enrollOktaVerifyPage.getErrorBox();
+  await t.expect(errorBox.innerText).contains(enableBiometricsMessage);
+  const errorTitle = enrollOktaVerifyPage.getErrorTitle();
+  await t.expect(errorTitle.innerText).contains(enableBiometricsMessageTitle);
+  await t.wait(4000);
+  await t.expect(logger.count(
+    record => record.response.statusCode === 200 &&
+    record.request.url.match(/poll/)
+  )).eql(1);
+  const successPage = new SuccessPageObject(t);
+  const pageUrl = await successPage.getPageUrl();
+  await t.expect(pageUrl)
+    .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
```

Comment:
Considering the IDX flow is based on individual responses and not the state, does it make sense to simplify these tests to skip success call test and save 4s every time ?  cc @haishengwu-okta

Reply:
The success helps test that when the server responds with the different response we do actually re-direct form this screen correctly.

Reply:
we can refactor if time is an issue